servicetalk-opentelemetry-http/src/main/java/io/servicetalk/opentelemetry/http/ServicetalkHttpClientAttributesGetter.java

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright © 2023 Apple Inc. and the ServiceTalk project authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.servicetalk.opentelemetry.http;
+
+import io.servicetalk.http.api.HttpRequestMetaData;
+import io.servicetalk.http.api.HttpResponseMetaData;
+import io.servicetalk.transport.api.HostAndPort;
+
+import io.opentelemetry.instrumentation.api.instrumenter.http.HttpClientAttributesGetter;
+
+import java.util.Collections;
+import java.util.List;
+import javax.annotation.Nullable;
+
+final class ServicetalkHttpClientAttributesGetter
+    implements HttpClientAttributesGetter<HttpRequestMetaData, HttpResponseMetaData> {
+
+    static final ServicetalkHttpClientAttributesGetter INSTANCE =
+        new ServicetalkHttpClientAttributesGetter();
+
+    private ServicetalkHttpClientAttributesGetter() {
+    }
+
+    @Override
+    public String getHttpRequestMethod(HttpRequestMetaData httpRequestMetaData) {
+        return httpRequestMetaData.method().name();
+    }
+
+    @Override
+    public List<String> getHttpRequestHeader(HttpRequestMetaData httpRequestMetaData, String name) {
+        CharSequence value = httpRequestMetaData.headers().get(name);
```

Comment:
This is incorrect, HTTP allows multiple values for the same header name. You should use `headers().values(name)` instead. 
Please add a test that verifies a case with multiple values.

Reply:
ok, refactored this.